pkg/action/install.go

```diff
@@ -716,5 +724,10 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 	if version != "" {
 		atVersion = fmt.Sprintf(" at version %q", version)
 	}
-	return filename, errors.Errorf("failed to download %q%s (hint: running `helm repo update` may help)", name, atVersion)
+
+	hint := "running `helm repo update` may help"
```

Comment:
pkg/action should not assume a CLI

Reply:
Note the existing error message assuming CLI. Should we just remove the hint entirely?

Reply:
resolved in 3e17c1a7efe7396b6c9cda5dbdb1a46a759e4b91